rules/sublime_rules/sublime_mailboxes_deactivated.yml

```diff
@@ -0,0 +1,160 @@
+AnalysisType: rule
+Description: A Sublime User disabled some mailbox(es).
+DisplayName: "Sublime Mailbox Deactivated"
+Enabled: true
+Filename: sublime_mailboxes_deactivated.py
+Runbook: Assess if this was done by the user for a valid business reason. Be vigilant to re-enable the mailboxes if it's in the best security interest for your organization's security posture.
+Reference: https://docs.sublime.security/docs/add-message-source
+Severity: Medium
+DedupPeriodMinutes: 60
+AlertTitle: Sublime message mailbox(es) were deactivated
+LogTypes:
+  - Custom.Sublime.AuditLogs
+RuleID: "Sublime.Mailbox.Deactivated"
+Threshold: 1
+Reports:
+  MITRE ATT&CK:
+    - TA0005:T1562.001 # Impair Defenses: Disable or Modify Tools
+Tests:
+  - ExpectedResult: false
+    Name: Other Events
+    Log:
```

Comment:
This is the old format of the logs from before we rolled out the custom parser, this events field is unrolled and actually each entry in the `events` field is a separate log event in Panther.